content/rs/references/crdb-cli-reference.md

```diff
@@ -0,0 +1,284 @@
+|`with-module <module_name>`|text|The name of the module to add to the database|
+|`force`|no arguments|Increase the configuration version even if there are no configuration changes|
+<!-- |`default-db-config <options>`|text|Default database configuration options|
+|`default-db-config-file <filename>`|file path|Default database configuration options| -->
+
+### Flushing the data from an Active-Active database
+
+The `crdb flush` command deletes all data in all of the instances of an Active-Active database.
+This command requires the CRDB-GUID of the database.
+
+{{< warning >}}
+This is an irreversible command.
+If the data in your database is important, make sure you back it up before you flush the database.
+{{< /warning >}}
+
+To flush the data from an Active-Active database, run:
+
+```sh
+crdb-cli crdb flush --crdb-guid <crdb-guid>
+```
+
+### Deleting an Active-Active database
+
+The `crdb delete` command deletes the Active-Active database and its data.
+This command requires the CRDB-GUID of the database.
+
+To delete an Active-Active database, run:
+
+```sh
+crdb-cli crdb delete --crdb-guid <crdb-guid>
+```
+
+{{< warning >}}
+This is an irreversible command.
+If the data in your database is important, make sure you back it up before you delete the database.
+{{< /warning >}}
+
+## Active-Active Instance Operations
+
+Active-Active instance operations affect the placement of database instances on participating clusters, but do not change the configuration or data in the database.
+
+### Adding an instance to an Active-Active Database {#adding-an-instance-to-an-activeactive-database}
+
+The `crdb add-instance` command lets you add instances to an existing Active-Active database in order to host the database on additional clusters.
+This creates a read-write copy of the database on the specified cluster.
+
+When you add an instance to an Active-Active database, you must specify:
+
+|Flag and argument| Argument type| Description|
+|---|---|---|
+|`crdb-guid <guid-id>`| string| The ID of the Active-Active database that you want to add the instance to|
+|`instance fqdn=<cluster_fqdn>,username=<username>,password=<password>`| strings| The connection information for the participating cluster that will host the new instance|
+
+### Removing an instance from an Active-Active Database {#removing-an-instance-from-an-activeactive-database}
+
+The `remove-instance` command deletes all data from an Active-Active instance, deletes the instance from the participating cluster, and removes the instance from the list of instances for the Active-Active database.
+
+When you remove an instance from an Active-Active database, you must specify:
+
+|Flag and argument| Argument type| Description|
+|---|---|---|
+|`crdb-guid <guid-id>`| string| The ID of the Active-Active database that you want to add the instance to|
+|`instance-id=<instance_id>`| integer| The ID of the instance to remove from the Active-Active database|
+
+{{< note >}}
```

Comment:
Let's add to this note that this flow should be performed only when impossible to restore network.
We don't encourage this flow

Reply:
I think that's what the note says. Can we leave it?